### PR TITLE
Translate 2018-04-25-open-source-charts-2017.md

### PR DESCRIPTION
**Pull request by yixin21**
zh-trans 2018-04-25-open-source-charts-2017.md

ref:k8smeetup/website-tasks#3242

### CODE REVIEW

**Comment by zacharysarah** [comment]
@yixin21 Please assign another Chinese localization contributor for review:

https://github.com/kubernetes/website/blob/c8c2f69f9bec0b6e13fd331a6ce74acd2f8cbe9f/OWNERS_ALIASES#L161

/unassign
cc @kubernetes/sig-docs-zh-reviews

File: content/zh/blog/_posts/2018-04-25-open-source-charts-2017.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+![Most Discussed](/images/blog-logging/2018-04-24-open-source-charts-2017/most-discussed.png)
+
+Of all the open source repositories on GitHub, none received more issue comments than [kubernetes/kubernetes](https://github.com/kubernetes/kubernetes/). [OpenShift](http://openshift.com/), a [CNCF certified distribution of Kubernetes](https://www.cncf.io/announcement/2017/11/13/cloud-native-computing-foundation-launches-certified-kubernetes-program-32-conformant-distributions-platforms/), took second place.  
+
+Open discussion with ample time for community feedback and review helps build shared infrastructure and establish new standards for cloud native computing.
+--->
+## GitHub 上讨论最多的
+
+2017 年讨论最多的两个回购协议都基于 Kubernetes：
```
Comment (tengqm) [inline content/zh/blog/_posts/2018-04-25-open-source-charts-2017.md:49]:
```suggestion
2017 年讨论最多的两个仓库都是基于 Kubernetes 的：
```

**Pull request merged**